scout/server/utils.py

```diff
@@ -106,6 +106,49 @@ def institute_and_case(store, institute_id, case_name=None):
     return institute_obj
 
 
+def rank_score(variant, rank_model):
+    """Calculate rank ranges for the variant and its corresponding rank model."""
+
+    def get_scores(field):
+        return [
+            int(v.get("score"))
```

Comment:
score are always ints, or? :D

Reply:
Or else the lovely ValueError exception crash!

Reply:
I wrapped the whole thing. :D Seems lazy but nice. Also added an error (warning) message to the rank score results view if it fails for some reason. I mean it should not, hence I think it would be overkill to make it fail in a more granular way (i.e. have it show results but not ranges).

<img width="1532" alt="image" src="https://user-images.githubusercontent.com/757478/153038555-ac45ef2f-89c7-4ac7-affa-d7f8263c5c98.png">

Reply:
the same thing would shown if the `.ini` file could not be fetched/parsed